### ISSUE TITLE
Any progress on this project?

### ISSUE THREAD

**Issue by cyberbit** [dlareau/KTANE-physical#1]
This is a neat project! A hardware version of KTANE is something I have been considering doing ever since watching the game for the first time many years ago. Are you still actively developing this? Any hints to the design of the hardware modules? I'd love to see a real working version!

**Comment by dlareau** [comment]
Yeah, I took a long break on it, but I am actually just starting back up on the next version.
I don't know which method you stumbled upon this repo through, but if it wasn't the imgur album or my website, you can see the photos of V1 here: https://imgur.com/a/J6HdM0v#w1oaDi1 or here: http://www.dillonlareau.com/projects/KTANE_physical

Basically all of the hardware for version 1 was point to point soldering, arduino nanos, and some stripboard where needed. (here is a photo of the back of 4 modules: https://photos.app.goo.gl/ooWz5mRqGPEx7vju6) Version 2 that I'm working on now will have PCBs, the design files for which will be included here once designed. 

Version 1 was also very fragile, in that while things were swappable, yanking on a module would likely destroy it. Version 2 will also greatly improve upon the plug and play nature of the modules, making it so that anybody can swap out modules without the worry of breaking something. 

If you have any other questions I'd be happy to chat.

**Comment by cyberbit** [comment]
Thanks for the reply! That looks slick.

A part of me wonders about the feasibility of adding "parameterization" modules (battery slots, indicator lights, etc.) that are permanent around the edges of the device. I've got lots of other ideas spinning in my head but I can't nail them down.

**Comment by dlareau** [comment]
Version 2 is already a bit ahead of you :)

While not quite in line with the original game layout, those 3 left-side boxes are for swappable items such as batteries, indicators, and ports. 
<img width="951" alt="Screen Shot 2019-12-13 at 11 44 43 AM" src="https://user-images.githubusercontent.com/2934378/70816930-7bf65180-1d9e-11ea-8d9e-eb5410b62203.png">
<img width="926" alt="Screen Shot 2019-12-13 at 11 43 58 AM" src="https://user-images.githubusercontent.com/2934378/70816937-7f89d880-1d9e-11ea-9399-5d0376c6f573.png">

**Comment by olbrichj** [comment]
This really looks neat! I was looking at the game for my gaming round. Searching for something more immersive I found this.

Some remarks/ideas:
- Will there be a serialnumber display at the side?
- Is there an option to connect multiple bombs together?
- In your images the modules looked huge, is this intentional?
- maybe instead of a full box how about creating one basic strip where you attach modules. This way you could add more than the above suggested 6 + 3.

**Comment by dlareau** [comment]
Thanks!

Replies in order:
- Serial number on the side was harder, so I ended up putting it in the extra space of the master/timer module, you can see it in [the gallery](https://imgur.com/a/J6HdM0v#w1oaDi1).
- There will be the option to connect multiple full units together, because of the way I've decided to architect things, the whole system runs off of a 4 wire bus that provides power and data, meaning hooking up a whole second case would be as easy as connecting those 4 wires between two cases. What connector that uses and where it will be haven't been sorted out yet.
- This project was meant to mimic the game and it's feel as much as possible. To that end, the modules were made to be in scale of the game by comparing against things like the ports/batteries on the side of the case. Plus, I can say from experience that things turned out well, when playing all of the elements felt accurate and very fun to interact with. Lastly, turns out I needed a lot of that space for electronics anyway, so it was all good.
- Similar to the above point, I was aiming for accuracy, so the 2x3 layout of the game felt right. The good news is that v2 is much more modular and includes 3d prints for both the module and the "module shell" which the module nestles into, so if someone else wanted to it would be really easy to just design a different holder for a bunch of module shells. Plus, I always imagined that if you wanted more than the 6+3, you could just hook up another case like talked about above.

**Comment by rubenvt** [comment]
Great job done! I love this project and think the V2 with PCB would be great, because I have no idea how I would accomplish making V1 at this moment without a good diagram for each module to know where to connect wires to and which components to use :-) if you have any details about it, it would make my day. keep up the good work
